pallets/loan/src/loan_type.rs

```diff
@@ -121,6 +121,8 @@ where
 			self.expected_loss_over_asset_maturity.is_positive(),
 			// maturity date should always be in future where now is at this instant
 			self.maturity_date > now,
+			// advance rate should not be more than 1
```

Comment:
Actually, in theory we could allow someone to loan more than the value at some point (i.e. undercollateralized loans).

Reply:
we can always remove this check when we are there. or is this something we are looking from the start ?